R.md

```diff
@@ -0,0 +1,68 @@
+# R on Graviton
```

Comment:
What is specific new info for Graviton?

Reply:
@RamaMalladiAWS #1 confirms for the R user that they can install it the same way.  #2 confirms that even though they may have heard that "Graviton is different", the install process is generally the same way.  It also walks through the solution to one known issue on Graviton when installing devtools.  #3 gives them a quick heads up on other possible Gravition related issues, specifically compiled code.

Reply:
I get the context now. So, in summary, you are saying "libgit2-devel" isn't available on ARM and hence one needs to build it explicitly and then install it for R.